### DISCUSSION TITLE
MySQL Migration

### DISCUSSION THREAD

**Discussion by xIvqn** [CodeURJC-DAW-2022-23/webapp9#52] [Announcements]
We've successfully migrated the database from H2 to MySQL. On merging, a MySQL server must be running on `localhost:3306` with the schema `tripscanner`. 
As for now, database is stated as `create-drop` to simulate an in-memory one but on release it might be changed to `none`.

It's pending creating a user without permissions to modify the schema.

### Installing MySQL database

Using Linux (Ubuntu):
```bash
sudo apt-get install mysql-server
```

Using Docker:
```bash
sudo docker run --rm -e MYSQL_ROOT_PASSWORD=password -e MYSQL_DATABASE=tripscanner -p 3306:3306 -d mysql:8.0.22
```

It's recommended using the Docker Container.

#### Installing workbench
```bash
sudo apt-get install mysql-workbench
mysql-workbench
```

**Comment by anna-trofimova** [comment]
MacOs:`sudo docker run --rm -e MYSQL_ROOT_PASSWORD=password -e MYSQL_DATABASE=tripscanner -p 3306:3306 -d arm64v8/mysql:8.0 `
